package.json

```diff
@@ -47,7 +47,9 @@
     "build": "napi build --platform --release",
     "test": "./node_modules/.bin/mocha test/unit -R spec -t 5000 --recursive",
     "unit": "./node_modules/.bin/mocha test/unit -R spec -t 5000 --recursive",
-    "examples": "node ./examples/DataStax/runner.js",
+    "ds-examples": "node ./examples/DataStax/runner.js",
+    "integration_short": "./node_modules/.bin/mocha test/integration/short -R spec -t 5000 --recursive",
+    "integration_long": "./node_modules/.bin/mocha test/integration/long -R spec -t 5000 --recursive",
```

Comment:
What are these parameters and why such values for them? Please document them in a comment.

Reply:
Especially with the removal of ``-R spec`` parameter, I'm not sure there is need to explain them

Reply:
I guess `recursive` runs all tests present in subdirectories.
`-t` is most probably a timeout, but what is the unit there? Seconds? If so, it's quite large. Milliseconds? Then it's very tight.

Reply:
I forgot that some of the test specify their own time. I'l look whether -t is even needed here.

Reply:
> Especially with the removal of `-R spec` parameter, I'm not sure there is need to explain them

Hmmm, it must have been lost in the changes I made to the history, so I reintroduced this change